src/components/CryptCard/styles.ts

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components/native';
+
+export const PercentChangeView = styled.View`
+  display: flex;
+  flex-direction: row;
+  gap: 3px;
+  align-items: center;
+`;
+
+export const PercentChangeNumber = styled.Text<{
+  negative: boolean;
+}>`
+  color: ${props => (props.negative ? 'red' : 'green')};
```

Comment:
check for all the places you should use theme, also styled components can use a theme, 
https://styled-components.com/docs/advanced

Reply:
I have now set up a custom light and dark theme, although it currently only changes background and font color. You want every color to be predefined in a theme? Or what other places are you referring to?